locales/base.json

```diff
@@ -50,7 +50,8 @@
     "toucan": "Tokenized carbon credits for climate action at scale",
     "cred": "Get your free web3 Credit Score",
     "kolor": "Access for conservation of Patagonian lands from everywhere",
-    "swftpro": "Bridge assets onto BTC/ETH/BSC/CELO/AVAX/+ safely and securely"
+    "swftpro": "Bridge assets onto BTC/ETH/BSC/CELO/AVAX/+ safely and securely",
+    "gooddollar": "Get free crtypo universal basic income. Stake, save and earn while supporting UBI"
```

Comment:
We have a few unpublished guidelines on description copy (we're working to share them).  Does this phrasing work for you?

> Claim UBI daily or stake to earn and support UBI